### DISCUSSION TITLE
./configure --with-heic is not configuring to build in heic support

### DISCUSSION THREAD

**Discussion by mreiland** [ImageMagick/ImageMagick#2008] [General]
I'm not sure what I'm missing.  I even went so far as to compile and make install the libheif library in the hopes that imagemagick would pick it up.

Does anyone have any more details on what is required to successfully build in heic support?

below is the libheif project I compiled and installed
https://github.com/strukturag/libheif

**Comment by dlemstra** [comment]
I think it really depends on which operating system you would want to run it. I have a project where I also build libheif for ImageMagick on Ubuntu and it seems that I have added `--prefix=/usr/local`. Maybe that helps you?
